src/targets/gpu/compile_gen.cpp

```diff
@@ -298,7 +317,7 @@ std::string generate_reduce(module m, const std::string& name)
     auto f        = g.generate_module(m, [&](instruction_ref ins, const auto& names) {
         if(contains(ins->name(), "reduce"))
         {
-            return reduce_op::generate(ins, names.at(ins->inputs().front()));
+            return reduce_op::generate(ins, cpp_generator::to_args(ins->inputs(), names));
         }
         else if(ins->name() == "pointwise")
         {
```

Comment:
Why is this else case empty? Remove this if not needed

Reply:
What else case is empty?

Reply:
Disregard refreshed github didn't load correctly on this for some reason